src/plugins/event_annotation/public/components/group_editor_controls/group_editor_controls.tsx

```diff
@@ -0,0 +1,184 @@
+import { css } from '@emotion/react';
+import type { DataView, DataViewSpec } from '@kbn/data-views-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { SavedObjectsTaggingApiUiComponent } from '@kbn/saved-objects-tagging-oss-plugin/public';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { QueryInputServices } from '@kbn/visualization-ui-components/public';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { EVENT_ANNOTATION_APP_NAME } from '../../../common/constants';
+import { EventAnnotationConfig, EventAnnotationGroupConfig } from '../../../common';
+import { AnnotationEditorControls } from '../annotation_editor_controls';
+import { AnnotationList } from './annotation_list';
+
+export const ENABLE_INDIVIDUAL_ANNOTATION_EDITING = false;
+
+export const GroupEditorControls = ({
+  group,
+  update,
+  setSelectedAnnotation: _setSelectedAnnotation,
+  selectedAnnotation,
+  TagSelector,
+  dataViews: globalDataViews,
+  createDataView,
+  queryInputServices,
+}: {
+  group: EventAnnotationGroupConfig;
+  update: (group: EventAnnotationGroupConfig) => void;
+  selectedAnnotation: EventAnnotationConfig | undefined;
+  setSelectedAnnotation: (annotation: EventAnnotationConfig) => void;
+  TagSelector: SavedObjectsTaggingApiUiComponent['SavedObjectSaveModalTagSelector'];
+  dataViews: DataView[];
+  createDataView: (spec: DataViewSpec) => Promise<DataView>;
+  queryInputServices: QueryInputServices;
+}) => {
+  // save the spec for the life of the component since the user might change their mind after selecting another data view
+  const [adHocDataView, setAdHocDataView] = useState<DataView>();
+
+  useEffect(() => {
+    if (group.dataViewSpec) {
+      createDataView(group.dataViewSpec).then(setAdHocDataView);
+    }
+  }, [createDataView, group.dataViewSpec]);
+
+  const setSelectedAnnotation = useCallback(
+    (newSelection: EventAnnotationConfig) => {
+      update({
+        ...group,
+        annotations: group.annotations.map((annotation) =>
+          annotation.id === newSelection.id ? newSelection : annotation
+        ),
+      });
+      _setSelectedAnnotation(newSelection);
+    },
+    [_setSelectedAnnotation, group, update]
+  );
+
+  const dataViews = useMemo(() => {
+    const items = [...globalDataViews];
+    if (adHocDataView) {
+      items.push(adHocDataView);
+    }
+    return items;
+  }, [adHocDataView, globalDataViews]);
+
+  const currentDataView = useMemo(
+    () => dataViews.find((dataView) => dataView.id === group.indexPatternId) || dataViews[0],
+    [dataViews, group.indexPatternId]
+  );
+
+  return !selectedAnnotation ? (
+    <>
+      <EuiText
+        size="s"
+        css={css`
+          margin-bottom: ${euiThemeVars.euiSize};
+        `}
+      >
+        <h4>
+          <FormattedMessage id="eventAnnotation.groupEditor.details" defaultMessage="Details" />
+        </h4>
+      </EuiText>
```

Comment:
Would it be better to use a `EuiTitle` component here (with `size="xs"`) instead of `EuiText`?